homeassistant/components/bluetooth_le_tracker/device_tracker.py

```diff
@@ -42,7 +49,12 @@ def handle_stop(event):
 
     hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, handle_stop)
 
-    def see_device(address, name, new_device=False):
+    if config.get(CONF_TRACK_BATTERY):
```

Comment:
There's no config schema that validates the new config. There should be a `PLATFORM_SCHEMA`.

Reply:
@guillempages Would you please update the schema in a fresh PR

Reply:
https://github.com/home-assistant/core/pull/35506